_posts/2019-05-17-nas-Reproduce-and-Replicate-in-Science.markdown

```diff
@@ -0,0 +1,20 @@
+---
+layout: post
+title: "New Report Examines Reproducibility and Replicability in Science, Recommends Ways to Improve Transparency and Rigor in Research."
+author: ehsteve
+---
+
+A new report by the National Academy of Sciences has issued new recommendations to improve confidence in science
+through enhancing Reproducibility and Replicability. Often used interchangeably, these two terms are defined
+precisely in the report. Reproducibility means obtaining consistent results through computation with the the same 
+input data while replicability means obtaining consistent results across experiments with their own data.
+
```

Comment:
I would also reference the [NAS report on Open Source Software Policy Options for Earth and Space Sciences](https://www.nap.edu/catalog/25217/open-source-software-policy-options-for-nasa-earth-and-space-sciences). It recommends the following: 

> NASA Science Mission Directorate should explicitly recognize the scientific value of open source software and incentivize its development and support, with the goal that open source science software becomes routine scientific practice.